docs/release/controllers.md

```diff
@@ -0,0 +1,181 @@
+
+In effect, this means a new minor version will at least be released for all Flux controllers approximately every four months, following each Kubernetes minor version release. To properly validate the controllers against the latest Kubernetes version, we typically allocate a time window of at least two weeks for end-to-end testing of Flux controllers.
+
+It is worth noting that in certain scenarios where project dependencies are not in sync with the Kubernetes version or conflicts arise, this two-week timeframe may prove insufficient, requiring additional time to address the issues appropriately.
+
+### Major releases
+
+Major releases are intended for drastic changes in the controller behaviour or security stance.
+
+A controller major release will be announced ahead of time throughout all communication channels,
+and a support window of one year will be provided for the previous major version.
+
+## Release cadence
+
+Flux controllers are at least released at the same rate as Kubernetes, following their cadence of three
+minor releases per year. After each Kubernetes minor release, all controllers are tested against the latest
+Kubernetes version and are released at approximately two weeks after Kubernetes.
+The newly released controllers offer support for Kubernetes N-2 minor versions.
+
+A Flux controller may have more than three minor releases per year, if maintainers decide to ship a 
+new feature or optimisation ahead of schedule.
+
+## Supported releases
+
+For Flux controllers we support the last three minor releases.
+
+Security fixes may be back-ported to those three minor versions as patch releases,
+depending on severity and feasibility.
+
+Note that back-porting is provided by the community on a best-effort basis.
+
+## Release artifacts
+
+Each controller release produces the following artifacts:
+
+- Source code (GitHub Releases page)
+- Software Bill of Materials in SPDX format (GitHub Releases page)
+- Kubernetes manifests such as CRDs and Deployments (GitHub Releases page)
+- Signed checksums of source code, SBOM and manifests (GitHub Releases page)
+- Multi-arch container images (GitHub Container Registry and DockerHub)
+
+All the artifacts are cryptographically signed and can be verified with Cosign.
+
+The release artifacts can be accessed based on the controller name and version.
+
+To import or update a controller's API package in a Go project:
+
+```shell
+go get github.com/fluxcd/<controller-name>/api@<version>
+```
+
+To verify and pull a controller's container image:
+
+```shell
+cosign verify ghcr.io/fluxcd/<controller-name>:<version>
+docker pull ghcr.io/fluxcd/<controller-name>:<version>
+```
+
+To download a controller's Kubernetes Custom resource definitions:
+
+```shell
+curl -sL https://github.com/fluxcd/<controller-name>/releases/download/<version>/<controller-name>.crds.yaml
+```
+
+## Controller release procedure
```

Comment:
I would move this to a separate document (e.g. `process.md`), as the audience is different.